materialize-postgres-rc/.snapshots/TestSQLGeneration

```diff
@@ -80,8 +80,8 @@ EXECUTE load_1--- End "Delta Updates" execLoadInsert ---
 SELECT 0, r.flow_document
 	FROM flow_temp_table_0 AS l
 	JOIN "a-schema".target_table AS r
-		 ON  l.key1 is not distinct from r.key1
-		 AND l."key!2" is not distinct from r."key!2"
+		 ON  l.key1 = r.key1
```

Comment:
Can you update the test fixture to exercise the nullable case? While a bit gross, it can be done by manually modifying the MaterializationSpec after it's been loaded and adding "null" to inferred type set of a primary key location.